docs/transform.md

```diff
@@ -54,6 +54,66 @@ This is more efficient and recommended if the data is either:
 
 Please note the [`lines`](../readme.md#lines) option is unrelated: it has no impact on transforms.
 
+## Newlines
+
+Unless [`{transform, binary: true}`](#binary-data) is used, the transform iterates over lines.
+By default, newlines are stripped from each `line` argument.
+
+```js
+// `line`'s value never ends with '\n'.
+const transform = function * (line) { /* ... */ };
+
+await execa('./run.js', {stdout: transform});
+```
+
+However, if a `{transform, newline: true}` plain object is passed, newlines are kept.
```

Comment:
`{newline: true}` reads to me like it enables splitting on newlines. Maybe `preserveNewlines`?

Reply:
Yes, that's much clearer. Fixed in 04345af7c848b7a319e075fa81836e051316a20b.